### PR TITLE
CEDS-1695 Showing data field for errors in rejected notifications screen

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: test/views/RejectedNotificationErrorsViewSpec.scala [hunk 1]
```diff
@@ -51,31 +52,48 @@ class RejectedNotificationErrorsViewSpec extends UnitViewSpec with Stubs with In
...
     "have correct title" in {
 
-      view.getElementById("title").text() mustBe messages("rejected.notification.title")
+      view().getElementById("title").text() mustBe messages("rejected.notification.title")
     }
 
     "have correct back link" in {
 
-      val backLink = view.getElementById("link-back")
+      val backLink = view().getElementById("link-back")
```
Comment (bambuchaAdm) [inline test/views/RejectedNotificationErrorsViewSpec.scala]: can we extract default arg view render as variable and don't render it for each case ?

File: app/models/Pointer.scala [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      case Failure(_) => s
+    }
+  }.mkString(".")
+
+  override def toString: String = value
+}
+
+object Pointer {
+  implicit val format: Format[Pointer] = Format(
+    Reads(js => js.validate[JsString].map(string => Pointer(string.value))),
```
Comment (bambuchaAdm) [inline app/models/Pointer.scala]: Why not validate just asString ?

File: app/models/Pointer.scala [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    Writes(pointer => JsString(pointer.value))
+  )
+
+  def apply(value: String): Pointer = Pointer(value.split("\\.").toList)
```
Comment (bambuchaAdm) [inline app/models/Pointer.scala]: Please use option with `Char` not pattern based

File: test/views/RejectedNotificationErrorsViewSpec.scala (hunk 1 above)
Comment (PatrykRudnicki) [reply test/views/RejectedNotificationErrorsViewSpec.scala]: yup

File: app/models/Pointer.scala (hunk 3 above)
Comment (PatrykRudnicki) [reply app/models/Pointer.scala]: don't know what you mean

File: test/views/RejectedNotificationErrorsViewSpec.scala (hunk 1 above)
Comment (ed0906) [reply test/views/RejectedNotificationErrorsViewSpec.scala]: Whats the harm?

File: app/models/Pointer.scala (hunk 3 above)
Comment (bambuchaAdm) [reply app/models/Pointer.scala]: Sry i miss this with `replace(Char)` - split does not have split

File: test/views/RejectedNotificationErrorsViewSpec.scala (hunk 1 above)
Comment (bambuchaAdm) [reply test/views/RejectedNotificationErrorsViewSpec.scala]: Rendering 10x allocates big string 10x and puts unnecessary GC pressure on tests. Unless we expect that view render with same parameters give use different outcomes.

File: app/models/Pointer.scala (hunk 2 above)
Comment (PatrykRudnicki) [reply app/models/Pointer.scala]: Changed

**Pull request merged**